studio/LonaStudio/Workspace/WorkspaceViewController.swift

```diff
@@ -84,7 +84,11 @@ class WorkspaceViewController: NSSplitViewController {
     public var activePanes: [WorkspacePane] {
         get {
             return WorkspacePane.all.filter {
-                return !(splitViewItem(for: $0)?.isCollapsed ?? true)
+                if $0 == .bottom {
+                    return false
```

Comment:
Since I don't have a public bottomItem, what should I do here?
Should I add a public getter for `isCollapsed` property of bottomItem in the `componentEditorViewController`, and use it here?

Reply:
Would it work to use `utilitiesViewVisible` for this, with a `get` and `set` block instead of a `didSet`? E.g.

```swift
public var utilitiesViewVisible: Bool {
  get { return bottomItem.isCollapsed }
  set { setBottomItemVisibility(to: newValue) }
}
```

Then we'd only need one public variable for collapsed state